README.md

```diff
@@ -28,6 +34,8 @@ $.ajax({
 ### Question 5
 In an AJAX POST request, what does the argument of the `.done()` callback - in the example below, that would be `data` - represent? (Hint: it's not the same as in Question 4.)
 
+The data argument here is the table of users in addition to whatever data was posted into the user table.
```

Comment:
In any AJAX request, the data argument is always the response from the server. For a POST request, the convention is that the response should be the object that was just POSTed. But it's not going to be the set of all users.